spec/support/features/session_helper.rb

```diff
@@ -70,7 +70,7 @@ def sign_in_and_2fa_user(user = user_with_2fa)
     end
 
     def user_with_2fa
-      create(:user, :signed_up, phone: '+1 (555) 555-5556', password: VALID_PASSWORD)
+      create(:user, :signed_up, phone: '+1 (555) 555-0000', password: VALID_PASSWORD)
```

Comment:
glad you asked!

the `Proofer::Vendor::Mock` class has some pre-defined failing values for testing purposes.
e.g. https://github.com/18F/identity-proofer-gem/blob/master/lib/proofer/vendor/mock.rb#L23

So if you enter `Bad` for first name, or a SSN that includes `6666`, or a phone number of `555-555-5555` or a financial account of `00000000` then it will trigger a failure.